po/Makefile

```diff
@@ -0,0 +1,33 @@
+APPNAME=kano-updater
+
+MSGLANGS=$(notdir $(wildcard *po))
+MSGOBJS=$(addprefix ../locale/,$(MSGLANGS:.po=/LC_MESSAGES/$(APPNAME).mo))
+
+.PHONY: clean messages
+
+build: $(MSGOBJS)
+
+update: $(MSGLANGS)
+
+clean:
+	rm -f messages.pot
+	rm -rf ../locale
+
+define run-xgettext
+xgettext -f PYPOTFILES -L Python --force-po -o messages.pot
+xgettext -f CPOTFILES -L C -k_ -kN_ -j -o messages.pot
+endef
+
+messages:
+	$(run-xgettext)
+
+messages.pot:
+	$(run-xgettext)
+
+%.po: messages.pot
+	msgmerge -N -U $*.po messages.pot
```

Comment:
I found the bug report [here](https://bugs.launchpad.net/qemu/+bug/668799) and it claims to have been fixed in June 2013 so you would have thought that we would already have the fix???